packages/generator-honeycomb/app/templates/src/client/_.client.js

```diff
@@ -1,38 +1,27 @@
-/* global document */
-
-<%_ if (includeReact) { _%>
-import React from 'react';
-import { render } from 'react-dom';
-import { AppContainer } from 'react-hot-loader';
+<%_ if (includePreact) { _%>
+/** @jsx h */
+import { h, render } from 'preact';
 import Example from './components/Example';
+<%_ } _%>
+import './client.css';
```

Comment:
At the moment we test the existence of this file only with `handlebars`. Move it from `handlebars` to the general section (`creates expected files`).